### PR TITLE
Replace unified-agent image with variable

### PR DESCRIPTION
**Pull request by Hoxo**

### CODE REVIEW

File: deployment/tf/modules/v2/allocator.tf [hunk 1]
```diff
@@ -244,6 +244,6 @@ resource "kubernetes_service" "allocator_service" {
       port = local.allocator-port
       target_port = local.allocator-port
     }
-    type = "LoadBalancer"
+    type = "ClusterIP"
```
Comment (ArtoLord) [inline deployment/tf/modules/v2/allocator.tf:247]: We need LoadBalancer here to access allocator from workers

File: deployment/tf/modules/v2/channel-manager.tf [hunk 2]
```diff
@@ -140,6 +140,6 @@ resource "kubernetes_service" "channel_manager_service" {
       port = local.channel-manager-port
       target_port = local.channel-manager-port
     }
-    type = "LoadBalancer"
+    type = "ClusterIP"
```
Comment (ArtoLord) [inline deployment/tf/modules/v2/channel-manager.tf:143]: We need LoadBalancer here to access channel-manager from workers

File: deployment/tf/modules/v2/whiteboard.tf [hunk 3]
```diff
@@ -111,11 +111,10 @@ resource "kubernetes_service" "whiteboard_service" {
   }
   spec {
     selector = local.whiteboard-labels
-    load_balancer_ip = yandex_vpc_address.whiteboard_public_ip.external_ipv4_address[0].address
     port {
       port = local.whiteboard-port
       target_port = local.whiteboard-port
     }
-    type = "LoadBalancer"
+    type = "ClusterIP"
```
Comment (ArtoLord) [inline deployment/tf/modules/v2/whiteboard.tf:118]: Whiteboard service must have public ip

**Review by ArtoLord — CHANGES_REQUESTED** [review]

File: deployment/tf/modules/v2/allocator.tf (hunk 1 above)
Comment (Hoxo) [reply deployment/tf/modules/v2/allocator.tf:247]: Yeah, it's just my mistake that I accidentally committed these changes into this branch. It was for testing purpose only. I'll get rid of them

**Comment by Hoxo** [comment]
I've removed all noted changes. The purpose of the rest is to provide more variability to the installation.

**Review by ArtoLord — APPROVED** [review]

**Pull request merged**